filter/auth_spi.go

```diff
@@ -0,0 +1,15 @@
+package filter
+
+import (
+	"github.com/apache/dubbo-go/common"
+	"github.com/apache/dubbo-go/protocol"
+)
+
+type Authenticator interface {
+	Sign(protocol.Invocation, *common.URL) error
+	Authenticate(protocol.Invocation, *common.URL) error
+}
+
+type AccesskeyStorage interface {
```

Comment:
why Accesskey here? But AccessKey in filter/filter_impl/auth/accesskey_storage.go

Reply:
I will put `Accesskey` in auth_spi.go